README.md

```diff
@@ -24,7 +24,8 @@
 </p>
 <br/>
 
-nix-bitcoin is a collection of Nix packages and NixOS modules for easily installing **full-featured Bitcoin nodes** with an emphasis on **security**.
+nix-bitcoin is a collection of Nix packages and NixOS modules for easily installing
+**full-featured Bitcoin nodes** with an emphasis on **security**.
```

Comment:
Hm, generally one sentence, one line seems fine and simple.

Reply:
Sorry, this was unintentional.
But in general I'd vote for treating markdown like normal source and break lines at max. 100 chars or so.

Reply:
> or so

that's what I mean :D sentence is easy to remember and to do consistently